mantidproject/mantidimaging
# Add 180 degree projection protection in operations window

Fixes #672 

To test:
- Load data with a 180 degree projection
- Open operations window
- Perform an operation on tomo stack ensure 180 degree also has it applied
- Perform an operation on all stacks and ensure the 180 degree has it applied
- Perform an operation on only the 180 degree stack and select cancel and ensure it didn't apply
- Perform an operation on only the 180 degree stack and select ok and ensure it did apply
